index.html

```diff
@@ -50,7 +53,158 @@ <h2 class="greeting greeting-text headline-box">
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">Multi-Post Stories <br />Gain+Glory</h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+      </article>
+
+      <!--About me section-->
+      <article class="about-me">
+        <h2 class="about-text">About me</h2>
+        <p class="about-me-intro">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
```

Comment:
Kindly ensure that the colour of the intro text matches the design. Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit. Doing so now will help you get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. 

Your design
![image](https://user-images.githubusercontent.com/20554583/185590673-db6d4a46-9104-4e72-8a03-acdbfc33aaa1.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185591211-77430870-fb13-46d5-ba9e-d4158ac81174.png)

Reply:
[Optional] Consider modifying the background image to display as it does on the figma design.